src/runtime/pkg/govmm/qemu/qemu.go

```diff
@@ -250,6 +239,7 @@ const (
 )
 
 // Object is a qemu object representation.
+// nolint: govet
```

Comment:
@fidencio By the way, do we need to deal with the problem that arises here, i.e. field alignment?

Reply:
At some point we may need to, indeed.

One thing to keep in mind is that this specific check was **never ever made** on GoVMM repo.  The reason we see this failure on our repo is because we, however, do check for that.

So, right now, we're not making anything worse, we're just using GoVMM as we've always been using it, without any change.

I also mentioned this in the commit itself:
```
govmm: Ignore govet checks, at least for now

govet checks have been ignored on govmm repo, but those are enabled on
kata-containers one.  So, in order to avoid failing our CIs let's just
keep ignoring the checks for the govmm structs and have an issue opened
for fixing it whenever someone has cycles to do it.

The important bit here is, we're not making anything worse that it
already is. :-)

Signed-off-by: Fabiano Fidêncio <fabiano.fidencio@intel.com>
```

Did I answer your question, @Bevisy?

Also, pleasel if that's something you'd be willing to work on, feel free to jump on it!  With the caveat that with the transition to the rust shim, GoVMM may become ... dead, in the near future.

Reply:
Thank you for your answer, I know what I need to do next :smile: